src/styles/catalog.scss

```
@@ -0,0 +1,160 @@
+$size-card-item: 200px;
+$background-color-main: #fff;
+$cost-color: #060B35;
+$price-color: #616070;
+$button-color: #00ACDC;
+
+
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, $size-card-item);
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 48px;
+
+  &__title {
+    color: $cost-color;
+    height: 36px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin: 40px 16px 0;
+  }
+
+  &__img {
+    object-fit: contain;
+    display: block;
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 0;
+  }
+
+  &__product-code {
+    color:  $price-color;
+    font-size: 10px;
+    line-height: 14px;
+    display: block;
+
+    margin: 0;
+    padding-left: 16px;
+    padding-top: 4px;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $size-card-item);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $size-card-item);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $size-card-item);
+  }
+
+  &__item {
+    width: $size-card-item;
+    box-sizing: border-box;
+    display: block;
+    margin: 0 auto;
+    background-color: $background-color-main;
+    border-radius: 5px;
+    border: 1px solid #F3F3F3;
+    padding-bottom: 16px;
+    transition: transform 0.3s;
+
+    &:hover {
+      transform: scale(1.2);
+
+      & .card__title {
+        color: #34568b;
+      }
+    }
+  }
+
+  &__rang {
+    display: flex;
+    width: 166px;
```

Need to fix it everywhere
```suggestion
    width: 100%;
```